blog/src/components/Featured/CardWide.jsx

```diff
@@ -0,0 +1,54 @@
+import images from "../../utils/images";
+import "../../styles/Card.scss";
+import PropTypes from "prop-types";
+
+
+const CardWide = ({ blogItemData }) => {
+    const MAX_LENGTH = 250;
+
+  return (
+    <div className="card-grid grid">
+      <div className="card-grid-img">
+        <img data-testid='FeaturedImg' src = {`${blogItemData.image}`} alt = "Featured image" className="object-fit-cover" />
+      </div>
+      <div className="card-grid-text">
+        <div>
+          <div className="badge badge-primary">featured</div>
+          <h2 data-testid='headingfeatured' className="title-lg title featured-title">
+            { blogItemData.title }
+          </h2>
+          
+            {blogItemData.content.length > MAX_LENGTH ?
+            (<p className="text-lg text" data-testid='featureSummary'>
+                    {`${blogItemData.content.substring(0, MAX_LENGTH)}...`}<a href="#">Read more</a>
+            </p>) 
+            :
+            <p data-testid='featureSummary'>{blogItemData.content}</p>
+            }
+        </div>
+
+        <div className="card-footer flex justify-between items-center">
+            <div className="writer-info grid">
+                <div className="info-avatar">
+                <img src = { `${blogItemData.authorImage }`} alt="" className="object-fit-cover" />
+                </div>
+                <div className="info-intro">
+                    <p className="intro-name text-base">{ blogItemData.author }</p>
+                    <div className="intro-verify flex items-center">
+                        <img src = { `${images.verify_icon_light}`} className="verify-icon" />
+                        <span className="verify-status text-sm text">Verified writer</span>
+                    </div>
```

Comment:
Good job adding the verified badge but it would be better to add the verified status on the blog object and here based on value of verified show the verified unverified badge.